tools/certgen/certgen.go

```diff
@@ -30,16 +30,16 @@ var (
 	// serviceAddresses is a list of all the HTTP hostname:port combinations for generating a TLS certificate.
 	// It appears that gRPC does not care about validating the port number so we only add the HTTP addresses here.
 	serviceAddressList = []string{
-		"om-backend:51505",
+		"open-match-backend:51505",
 		"om-demo:51507",
```

Comment:
why aren't these values changing.

Reply:
I have no idea where they come from. They're not part of the Helm chart generated code. I think they're used in the tutorials?

Reply:
Some of them come from https://github.com/googleforgames/open-match/blob/master/install/02-open-match-demo.yaml for example, which appears to be hand-maintained. I assume that will be cleaned up as part of #1244 cleaning up the openmatch-customize subchart.